Gemfile

```diff
@@ -26,4 +26,3 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
-
```

Comment:
Ok. 

How can I tell if the gem is necessary? It worked on localhost without it, when I realized that maybe I was overzealous in adding things.